jdl2makeflow

```diff
@@ -209,17 +209,31 @@ done
 PROTO=${ALIEN_JDL_OUTPUTDIR%%://*}
 [[ $PROTO != $ALIEN_JDL_OUTPUTDIR ]] || { PROTO=local; mkdir -p $ALIEN_JDL_OUTPUTDIR; }
 [[ $PROTO == local || $PROTO == root ]] || { echo "Output protocol $PROTO not supported;" exit 1; }
+workers=15
 pushd to_transfer
+  count=0
   while read FILE; do
-    for ((I=1; I<=5; I++)); do
-      ERR=0
-      echo "Transferring $FILE to $ALIEN_JDL_OUTPUTDIR (attempt $I/5)"
-      case $PROTO in
-        local) mkdir -p $(dirname "$ALIEN_JDL_OUTPUTDIR/$FILE"); cp -v $FILE $ALIEN_JDL_OUTPUTDIR/$FILE && break || ERR=$? ;;
-        root)  xrdcp -f $FILE $ALIEN_JDL_OUTPUTDIR/$FILE && break || ERR=$? ;;
-      esac
-    done
+    ln -nfs $FILE $tempdir/$count
```

Comment:
`$tempdir` is not set, don't use anything outside the job's workdir. Since you are in `to_transfer`, which is one level below the job's workdir, you can set it to:

```bash
STATEDIR=$(mkdir -p ../statedir; cd ../statedir && pwd)
```